kep/numa-nodes-scoring/kep.yaml

```diff
@@ -0,0 +1,12 @@
+title: Scoring nodes based on number of NUMA nodes required to run the pod.
```

Comment:
Can you also rename the folder to `454-numa-nodes-scoring`?

Reply:
Can the title be a bit concise like "NUMA aware scoring"?

Reply:
:+1: